core/controllers/domain_objects_validator.py

```diff
@@ -35,4 +36,20 @@ def validate_exploration_change(obj):
     """
     # No explicit call to validate_dict method is necessary, because
     # ExplorationChange calls it while initialization.
+
     exp_domain.ExplorationChange(obj) # type: ignore[no-untyped-call]
+
+
+def validate_new_config_property_values(obj):
+    # type: (Dict[Any, Any]) -> None
+    """Validates new config property values.
+
+    Args:
+        obj: dict. Data that needs to be validated.
+    """
+    for (name, value) in obj.items():
+        config_property = config_domain.Registry.get_config_property(name) # type: ignore[no-untyped-call] # pylint: disable=line-too-long
```

Comment:
```suggestion
        config_property = (
            config_domain.Registry.get_config_property(name)) # type: ignore[no-untyped-call]
```
Please avoid the usage of disabling pylint.

Reply:
@rohitkatlaa I have all possible ways, but long-line error was not fixed, please guide me.
Thank you.

Reply:
I think you should be able to break before name.